abi2specs

```diff
@@ -209,7 +209,7 @@ let behaviours = acts
       .join("\n")
 
     const storage_fail = "  " + Object.keys(act.storage && act.storage.ACCT_ID || [])
-      .map(key => key + " |-> (" + act.storage.ACCT_ID[key][0] + (act.storage.ACCT_ID[key].length > 1 ? " => _ " : "") + ")" )
+      .map(key => key + " |-> (" + act.storage.ACCT_ID[key][0] + (act.storage.ACCT_ID[key].length > 1 ? " => " + act.storage.ACCT_ID[key][0] : "") + ")" )
```

Comment:
this is not correct, should be "_"

Reply:
We want to enforce storage to be the same after running a failed spec.